doc/usecases/snippet_sharing/UC_convert_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Convert configuration snippet
+
+## Summary
+
+Title: Convert configuration snippet
+Scope: Configuration conversion
+Level: User Goal
+Actors: Anonymous user (+ all others)
+Brief: User can input a configuration snippet in any supported format (e.g. ini, xml) and convert it into another supported format (e.g. json).

```

Comment:
It is bad design to create a dummy snippet only for conversion and delete it again afterwards. The additional endpoint is no effort and already done.